recipes/tvm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "tvm" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/apache/incubator-tvm/archive/v{{ version }}.tar.gz
+  sha256: e9724ccb3e3bd4c1c48fa6b56450443d390329b02987465f58824d90337e090e
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake >=3.5
+    - ninja
+  host:
+    - python
+    - cython
+    - numpy
+    - setuptools
+    - llvm-tools
+    - llvmdev
```

Comment:
Is llvm used?

Reply:
I think just ``llvm-tools`` is used, so we ca remove ``llvmdev``